www/docs/en/dev/cordova/events/events.md

```diff
@@ -323,6 +323,16 @@ function onBackKeyDown() {
 }
 ```
 
+### Windows Quirks
+
+Throw an error in a `backbutton` callback to force the default behavior:
+
+```javascript
+document.addEventListener('backbutton', function (evt) {
+    throw "Exit"; // This will suspend the app
+}, false);
+```
+
```

Comment:
I don't see another way to handle this situation taking into account the cordova-js implementation.
Theoretically we could use a callback return value from `cordova.fireDocumentEvent('backbutton', evt, true);` but it only allows to capture an exception but not a value.